eclair-core/src/main/scala/fr/acinq/eclair/channel/InteractiveTxBuilder.scala

```diff
@@ -606,10 +616,25 @@ private class InteractiveTxBuilder(replyTo: ActorRef[InteractiveTxBuilder.Respon
       return Left(InvalidCompleteInteractiveTx(fundingParams.channelId))
     }
 
-    val minimumFee = Transactions.weight2fee(fundingParams.targetFeerate, minimumWeight)
-    if (sharedTx.fees < minimumFee) {
-      log.warn("invalid interactive tx: below the target feerate (target={}, actual={})", fundingParams.targetFeerate, Transactions.fee2rate(sharedTx.fees, minimumWeight))
-      return Left(InvalidCompleteInteractiveTx(fundingParams.channelId))
+    previousTransactions.headOption match {
+      case Some(previousTx) =>
+        // This is an RBF attempt: even if our peer does not contribute to the feerate increase, we'd like to broadcast
+        // the new transaction if it has a better feerate than the previous one. This is better than being stuck with
+        // a transaction that doesn't confirm.
+        val previousUnsignedTx = previousTx.tx.buildUnsignedTx()
+        val previousMinimumWeight = previousUnsignedTx.weight() + previousUnsignedTx.txIn.length * minimumWitnessWeight
+        val previousFeerate = Transactions.fee2rate(previousTx.tx.fees, previousMinimumWeight)
+        val nextFeerate = Transactions.fee2rate(sharedTx.fees, minimumWeight)
```

Comment:
We should probably compare to `Transactions.weight2fee(fundingParams.targetFeerate, minimumWeight)` and log the fact that the non-initiator didn't contribute to the feerate increase.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2402/commits/c9afb83fcdab18d91660b86a2d7307032db2ccc5, I added a log for the case where the remote peer contributed the exact same amount than the previous iteration.